templates/metrics/css/popularity.html

```diff
@@ -9,6 +9,14 @@
   {% include "metrics/css/_css_metric_nav.html" %}
 {% endblock %}
 
+{% block properties %}
+<nav class="data-panel" style="font-size: smaller;">
+  All Properties:<a href="/metrics/css/timeline/popularity" style="padding-left: 2%; padding-right: 2%;">Timeline</a> | 
+  <a href="/metrics/css/popularity"  style="padding-left: 2%; padding-right: 2%;">Stack Rank</a>
```

Comment:
For this link and the others, please remove the style attributes and add CSS rules instead.  You could put it in metrics.scss after .description.    Also, rather than padding as a %, try using var(--content-padding-quarter) or just 4px.

Reply:
Done